main.py

```diff
@@ -2,6 +2,52 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    for genre in genres:
+        Genre.objects.create(name=genre)
+
+    actors = [
+        ["George", "Klooney"],
+        ["Kianu", "Reaves"],
+        ["Scarlett", "Keegan"],
+        ["Will", "Smith"],
+        ["Jaden", "Smith"],
+        ["Scarlett", "Johansson"]
+    ]
+    for actor in actors:
+        first_name, last_name = actor
+        Actor.objects.create(first_name=first_name, last_name=last_name)
+
+    Genre.objects.filter(
+        name="Dramma"
+    ).update(name="Drama")
+
+    Actor.objects.filter(
+        first_name="George",
+        last_name="Klooney"
+    ).update(last_name="Clooney")
+
+    Actor.objects.filter(
+        first_name="Kianu",
+        last_name="Reaves"
+    ).update(first_name="Keanu", last_name="Reeves")
+
+    Genre.objects.filter(
+        name="Action"
+    ).delete()
```

Comment:
When there is no need for line breaking, you could place all arguments in one line
```suggestion
    Genre.objects.filter(name="Action").delete()
```